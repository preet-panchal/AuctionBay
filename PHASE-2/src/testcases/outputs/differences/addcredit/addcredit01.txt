--- ./../testcases/outputs/expected/addcredit/addcredit01.txt	2023-02-03 00:25:47
+++ ./../testcases/outputs/actual/addcredit/addcredit01.txt	2023-03-05 16:32:32
@@ -13,12 +13,15 @@
 
 Enter operation:
 Enter username:
-Hello admin01 (AA)!
 
+Error: This user does not exist in the user-accounts file.
 Enter operation:
-Enter the credit amount to add: 
-Enter the username to add the credit to:
-Error: The user does not exist in the user-accounts file.
+Error: You must login first.
 
 Enter operation:
-admin01 successfully logged out.
\ No newline at end of file
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
