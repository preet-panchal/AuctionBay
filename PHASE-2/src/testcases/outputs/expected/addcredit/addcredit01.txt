Welcome to AuctionBay!
--
Here is a list of operations (what to input):
- Login (login)
- Create (create)
- Delete (delete)
- Advertise (advertise)
- Bid (bid)
- Refund (refund)
- Add Credit (addcredit)
- List All Items (listall)
- Logout (logout)

Enter operation:
Enter username:
Hello User01 (AA)!

Enter operation:
Enter the username to add credit to:
Error: The user does not exist in the user-accounts file.

Enter operation:
User01 successfully logged out.
